nmrfx-structure/src/main/java/org/nmrfx/structure/chemistry/miner/PathIterator.java

```diff
@@ -267,23 +258,21 @@ void dumpPath() {
             int iAtom = getAtomIndex(i);
             System.out.print(iAtom + " ");
         }
-        System.out.println("");
+        System.out.println();
         for (int i = 0; i < n; i++) {
             IAtom atom = ac.getAtom(getAtomIndex(i));
             System.out.print(atom.getSymbol());
         }
-        System.out.println("");
+        System.out.println();
```

Comment:
These should probably be changed to log.debug, you can use String builder and log just once at the end

Reply:
Switched to logs (though still a method which uses StringBuilder and can return string, I call this from Python while working with new structures)